library/artifactory_repo.py

```diff
@@ -34,7 +34,7 @@
     repo:
```

Comment:
might be good to rename this to `name`... would match other modules a bit closer, since the module name is `artifactory_repo`

Reply:
That works.